index.html

```diff
@@ -53,5 +46,88 @@ <h3>I'am a software developer</h3>
             </div>
         </section>
     </div>
+
+    <section class="my-main-works">
+        <section class="recent-works">
+            <div class="works-title">
+                <h2>My Recent Works</h2>
+                <hr class="solid-line">
+            </div>
+        </section>
+        <section class="multi-project">
+            <div class="project-image"></div>
+            <h3 class="project-title">Multi-Post Stories</h3>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <div class="project-tag">
+                <p>CSS</p>
+                <p>html</p>
+                <p>bootstrap</p>
+                <p>Ruby</p>
+            </div>
+            
+             <div class="project-view">
+                <a href="#">See Project</a>
+            </div>  
+        </section>
+        <section class="single-project">
+            <div class="project-image">
+                <h3 class="single-title">Professional Art<br> Printing Data</h3>
+                <p class="single-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="single-language">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                
+            </div>
+            <div class="view-project">
+                <a href="#" id="view-project-id">See Project</a>
+            </div>
+        </section>
```

Comment:
- [ ] According to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=3CbGHNhPHWBiQT4V-0) in `My Recent Works` section there should be seven cards. You have used only two cards here; one is `Multi Post Stories,` and another is `Professional Art Printing Data`.
  - Kindly use more cards so that they match the Figma design, and you can get a template for future use.